hawkbit-mgmt-resource/src/test/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtSoftwareModuleResourceTest.java

```diff
@@ -457,7 +458,7 @@ public void invalidRequestsOnSoftwaremodulesResource() throws Exception {
                 .contentType(MediaType.APPLICATION_JSON)).andDo(MockMvcResultPrinter.print())
                 .andExpect(status().isBadRequest());
 
-        final SoftwareModule missingName = entityFactory.generateSoftwareModule(osType, null, "version 1", null, null);
+        final SoftwareModule missingName = new JpaSoftwareModule(osType, null, "version 1", null, null);
```

Comment:
why using here `new JpaDistributionSetType` and not like before the `entityFactory.generateDistributionSetType`?